frontend/src/views/Pages/AccountOverview/GddTransactionList.vue

```diff
@@ -7,7 +7,7 @@
           {{ $t('error.no-transactionlist') }}
         </small>
       </div>
-      <div v-if="!transactionCount" class="test-empty-transactionlist text-right">
+      <div v-if="transactionCount === 'undefined'" class="test-empty-transactionlist text-right">
```

Comment:
This seems to be a hack - cant we set null on the backend? @Mogge @Elweyn

Reply:
```
!0 -> true
!undefined -> true
!null -> true
```
but
```
!1 -> false
```
I really do not understand how this change could improve our code

Reply:
Furthermore
```
undefined === 'undefined' -> false
```

Reply:
Changed the logic so that the transactionCount is set to -1 if the backend throws an error.
So the check is now:
```
transactionCount < 0
```